web/app/scripts/views/record/add-edit-controller.js

```diff
@@ -97,10 +105,42 @@
             constantFieldsValidationErrors();
         }
 
+        /**
+         * Since the date and time pickers rely on the browser's local timezone with
+         * no way to override, we need to modify the occurred datetime before it gets
```

Comment:
Ugh. There are some pickers that use moment.js, such as the date/time picker and [this thing](https://www.npmjs.com/package/angular-bootstrap3-datepicker).